ProcessLib/ThermoMechanics/ThermoMechanicsFEM.h

```diff
@@ -227,14 +235,13 @@ class ThermoMechanicsLocalAssembler
             auto& eps_m_prev = _ip_data[ip].eps_m_prev;
             auto& state = _ip_data[ip].material_state_variables;
 
-            double const delta_T =
-                N.dot(T) - _process_data.reference_temperature;
+            double const dT = N.dot(T_dot) * dt;
             // calculate thermally induced strain
             // assume isotropic thermal expansion
             auto const alpha =
                 _process_data.linear_thermal_expansion_coefficient(
                     t, x_position)[0];
-            double const linear_thermal_strain = alpha * delta_T;
+            double const linear_thermal_strain = alpha * dT;
```

Comment:
Maybe a comment in the code would be appropriate that the linear thermal strain now is w.r.t. the state from the previous timestep and not w.r.t. a reference temperature (if I understood that correctly).

Reply:
Changed the name to `linear_thermal_strain_increment`.